plugins/aws/resoto_plugin_aws/resource/ecs.py

```diff
@@ -0,0 +1,1662 @@
+    image: Optional[str] = field(default=None)
+    image_digest: Optional[str] = field(default=None)
+    runtime_id: Optional[str] = field(default=None)
+    last_status: Optional[str] = field(default=None)
+    exit_code: Optional[int] = field(default=None)
+    reason: Optional[str] = field(default=None)
+    network_bindings: List[AwsEcsNetworkBinding] = field(factory=list)
+    network_interfaces: List[AwsEcsNetworkInterface] = field(factory=list)
+    health_status: Optional[str] = field(default=None)
+    managed_agents: List[AwsEcsManagedAgent] = field(factory=list)
+    cpu: Optional[str] = field(default=None)
+    memory: Optional[str] = field(default=None)
+    memory_reservation: Optional[str] = field(default=None)
+    gpu_ids: List[str] = field(factory=list)
+
+
+@define(eq=False, slots=False)
+class AwsEcsInferenceAccelerator:
+    kind: ClassVar[str] = "aws_ecs_inference_accelerator"
+    mapping: ClassVar[Dict[str, Bender]] = {"device_name": S("deviceName"), "device_type": S("deviceType")}
+    device_name: Optional[str] = field(default=None)
+    device_type: Optional[str] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsEcsEnvironmentFile:
+    kind: ClassVar[str] = "aws_ecs_environment_file"
+    mapping: ClassVar[Dict[str, Bender]] = {"value": S("value"), "type": S("type")}
+    value: Optional[str] = field(default=None)
+    type: Optional[str] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsEcsResourceRequirement:
+    kind: ClassVar[str] = "aws_ecs_resource_requirement"
+    mapping: ClassVar[Dict[str, Bender]] = {"value": S("value"), "type": S("type")}
+    value: Optional[str] = field(default=None)
+    type: Optional[str] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsEcsContainerOverride:
+    kind: ClassVar[str] = "aws_ecs_container_override"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "name": S("name"),
+        "command": S("command", default=[]),
+        "environment": S("environment", default=[]) >> ForallBend(AwsEcsKeyValuePair.mapping),
+        "environment_files": S("environmentFiles", default=[]) >> ForallBend(AwsEcsEnvironmentFile.mapping),
+        "cpu": S("cpu"),
+        "memory": S("memory"),
+        "memory_reservation": S("memoryReservation"),
+        "resource_requirements": S("resourceRequirements", default=[]) >> ForallBend(AwsEcsResourceRequirement.mapping),
+    }
+    name: Optional[str] = field(default=None)
+    command: List[str] = field(factory=list)
+    environment: List[AwsEcsKeyValuePair] = field(factory=list)
+    environment_files: List[AwsEcsEnvironmentFile] = field(factory=list)
+    cpu: Optional[int] = field(default=None)
+    memory: Optional[int] = field(default=None)
+    memory_reservation: Optional[int] = field(default=None)
+    resource_requirements: List[AwsEcsResourceRequirement] = field(factory=list)
+
+
+@define(eq=False, slots=False)
+class AwsEcsInferenceAcceleratorOverride:
```

Comment:
Looks like you can replace this type with `AwsEcsInferenceAccelerator`

Reply:
good catch!